backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/incremental/BorderNodePositionTests.java

```diff
@@ -71,6 +76,31 @@ void testSnap() {
         incrementalLayoutEngine.layout(Optional.empty(), initializeDiagram, new LayoutConfiguratorRegistry(List.of()).getDefaultLayoutConfigurator());
 
         this.checkBorderNodesAtInitialPosition(borderNodes);
+
+        this.checkBorderNodeLabel(borderNodes.get(0).getLabel(), BORDER_NODE_LABEL_TEXT_POSITION, BORDER_NODE_LABEL_TEXT_BOUNDS);
+    }
+
+    private void checkBorderNodeLabel(LabelLayoutData labelLayoutData, Position borderNodeLabelTextPosition, TextBounds borderNodeLabelTextBounds) {
+        assertThat(labelLayoutData.getPosition()).isEqualTo(borderNodeLabelTextPosition);
+        assertThat(labelLayoutData.getTextBounds()).isEqualTo(borderNodeLabelTextBounds);
+    }
+
+    @Test
+    void testBorderNodeCreationEvent() {
```

Comment:
visibility again

Reply:
it is adviced by checkstyle
JUnit5 test classes and methods should have default package visibility (java:S5786)

Reply:
I put the public visibility back